packages/now-static-build/src/frameworks.ts

```diff
@@ -514,6 +514,13 @@ const frameworkList: Framework[] = [
     buildCommand: 'bundle exec middleman build',
     getOutputDirName: async () => 'build',
   },
+  {
+    name: 'Nift',
+    slug: 'nift',
+    buildCommand: 'nift build-all',
```

Comment:
Where does the `nift` binary come from? Is it on https://www.npmjs.com or another package registry?

Reply:
Binaries are available with the release assets on GitHub, see https://github.com/nifty-site-manager/nsm/releases 

Each release will have a Vercel.zip asset containing `nift` and `nsm` binaries (they are identical to each other, but some users may override the build command and use `nsm`).

Might be worth chatting to Leo and Matt who know about this stuff as well.

Reply:
Thanks, I'll ask them 👍